MorseDecodeEncode/src/app/decode/decode.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { MorseCodeService } from '../morse-code.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class DecodeService {
+  constructor(private morseCordeService: MorseCodeService) {}
+
+  public decode(morseCodeInput: string) {
+    let result = '';
+    let morseCodeLetter = '';
+
+    // for (let char of morseCodeInput) {
```

Comment:
Why so much commented code? Try to remove unused code before checking in or at least add a comment to describe why code is commented out.